constitution.tex

```diff
@@ -520,9 +520,11 @@
 \asubsubsection{The Introductory Process}
 The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
 \asubsubsubsection{The Evaluation Period}
-The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
+The Process will occur either once or twice per semester, and will last for six (6) weeks.
```

Comment:
This allows for one intro process at the end of the semester... do we want that?